### PR TITLE
Clarify that in Python there is a week descriptor called 'fifth'

### PR DESCRIPTION
**Pull request by ksnortum**
Add a clarification to the docs that in Python there is a week descriptor called `fifth` that should be the fifth weekday of the month, if one exists.

### CODE REVIEW

File: exercises/practice/meetup/.docs/instructions.append.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Instructions append
 
+## How this Exercise is Structured in Python
+
+There is another week descriptor, `fifth`, for the fifth weekday of the month, if there is one.  If there is not, you should raise an exception.
```
Comment (BethanyG) [inline exercises/practice/meetup/.docs/instructions.append.md:5]:
```suggestion
We have added an additional week descriptor (`fifth`) for the fifth weekday of the month, if there is one.  
If there is not a fifth weekday in a month, you should raise an exception.
```

**Review by BethanyG — APPROVED** [review]
Thank you for this!  I've suggested a little rephrase, and edited to conform to our 'one sentence per line' [formatting rule](https://github.com/exercism/docs/blob/main/building/markdown/markdown.md#layout).

Absolutely fine if you don't like/accept the suggested rephrase.

**Comment by BethanyG** [comment]
Once you've looked this over (and approve or not), I will merge.  😄

**Comment by ksnortum** [comment]
Looks good to me.

**Pull request merged**